HW10/HW10.py

```diff
@@ -0,0 +1,6 @@
+def rec_n(n, start=1):
+    print(start)
+    if start < n: rec_n(n, start + 1)
```

Comment:
nit: better not to write in one line too many operations
```suggestion
    if start < n: 
        rec_n(n, start + 1)
```